### ISSUE TITLE
attempt to import missing "get_env_index" preventing mamba env create

### ISSUE THREAD

**Issue by teresa-recursion** [QuantStack/mamba#168]
First, thanks for open-sourcing this, using mamba is cutting our build times in half!

Second, apologies if you already know about this but we've been getting this error when we try to create an environment using `mamba env create - f environment.yml` for the last few days.

It looks like `mamba/mamba_env.py` is trying to import `get_env_index` from `utils.py` which existed in `0.1.2` but looks like was removed in `0.2.0`.
```
Traceback (most recent call last):
  File "/opt/conda/bin/mamba", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.7/site-packages/mamba/mamba.py", line 736, in main
    from mamba import mamba_env
  File "/opt/conda/lib/python3.7/site-packages/mamba/mamba_env.py", line 24, in <module>
    from mamba.utils import get_env_index, to_package_record_from_subjson
ImportError: cannot import name 'get_env_index' from 'mamba.utils' (/opt/conda/lib/python3.7/site-packages/mamba/utils.py)
```

I'm happy to put in a PR that just removes this import if that is helpful and thanks again for providing this for the community to use.

**Comment by wolfv** [comment]
Hi @teresa-recursion that might well be an issue ... we'll have a fix for this by tonight.

**Comment by wolfv** [comment]
I have merged a fix here #169 

I'll make a release tonight, or is it urgent?

**Comment by teresa-recursion** [comment]
It's not urgent- we are just having our build use the 0.1.2 release, no big deal.  Thank you for acting on this so quickly!

**Comment by wolfv** [comment]
this is fixed with 0.2.4 (and we have a test to prevent such issues for the future).

Thanks again!
